patch/armhf-sonic-kernel-config.patch

```diff
@@ -0,0 +1,97 @@
+From 3dd4829a62b201eb7091aaf31cc93634539e0ddd Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Wed, 25 Sep 2019 01:13:59 +0530
+Subject: [PATCH] armhf sonic kernel config
+
+---
+ debian/build/build_armhf_none_armmp/.config | 15 ++++++++++++++-
+ 1 file changed, 14 insertions(+), 1 deletion(-)
+
+diff --git a/debian/build/build_armhf_none_armmp/.config b/debian/build/build_armhf_none_armmp/.config
+index e2b7581..7229ca1 100644
+--- a/debian/build/build_armhf_none_armmp/.config
++++ b/debian/build/build_armhf_none_armmp/.config
+@@ -1501,6 +1501,7 @@ CONFIG_NET_ACT_POLICE=m
+ CONFIG_NET_ACT_GACT=m
+ CONFIG_GACT_PROB=y
+ CONFIG_NET_ACT_MIRRED=m
++CONFIG_NET_ACT_SAMPLE=m
```

Comment:
please include this in the kernel-enable-psample-and-act_sample-drivers.patch file.

Reply:
Right now I cannot add patch where it has diff for different arch .config files.
I have mentioned the reason in the above comment.